WordPress/Classes/ViewRelated/System/Notices/NoticeView.swift

```diff
@@ -258,16 +258,15 @@ class NoticeView: UIView {
     }
 
     private enum Metrics {
-        static let cornerRadius: CGFloat = 13.0
+        static let cornerRadius: CGFloat = 4.0
```

Comment:
I saw @nheagy comment on the corner radius not looking iOS like and I agree. I know that Muriel says 4.0 but thought we agreed to find a middle ground. Are we gonna keep it as 4.0 for now?

Reply:
@megsfulton said that we should keep it at 4.0 for now.